### ISSUE TITLE
Warn about mismatched Dockstore UI and webservice

### ISSUE THREAD

**Issue by denis-yuen** [ga4gh/dockstore#1308]
### Desired behaviour

Option 1) Automated check if ui2 tests are running against an out-of-date webservice
May need to use encrypted token, send message to funkbot or slack
Build messages channel
Option 2) Banner that shows up for developers (locally or in staging) if the ui version is expecting a webservice version that does not match (the UI essentially should know its version, if only through what version was used to generate swagger classes)

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/DOCK-327)
┆Issue Type: Story
┆Issue Number: DOCK-327

**Comment by denis-yuen** [comment]
Might be able to share code with #1321

**Comment by agduncan94** [comment]
What is an out-of-date webservice version. Does this refer to any release that is not the most recent?

**Comment by denis-yuen** [comment]
I think we settled on option 2 in the post mortem actually

**Comment by agduncan94** [comment]
As per the discussion this can be merged into a general banner that exists on staging to make it obvious that you are on staging.

This option could either be a field similar to launch firecloud, or we could use environment configs.

**Comment by agduncan94** [comment]
Depends on #1346

**Comment by agduncan94** [comment]
Making a note here that it seems like the version generation code for generating the versions.ts does not work if you run in sudo mode.

**Comment by denis-yuen** [comment]
Hmmm, the compose setup basically runs as root too.

**Comment by denis-yuen** [comment]
Ran into this with 1.5.2. hotfix release, need to revisit
